hammett/core/conversation_handler.py

```diff
@@ -63,6 +63,11 @@ async def handle_update(  # type: ignore[override]  # noqa:C901, PLR0912
         -------
             Object or None.
 
+        Raises
+        ------
+            ApplicationHandlerStop: If it is needed to prevent execution of any other handler
+            (even in different groups).
```

Comment:
```
            ApplicationHandlerStop: If it's necessary to prevent the execution of any other handler
            (even in different groups).
```